### DISCUSSION TITLE
[Feature Request] How to pass additional browser settings such as Zoom Level?

### DISCUSSION THREAD

**Discussion by RMichaelPickering** [Voltstro-Studios/UnityWebBrowser#271] [Requests]
### What platform are you experiencing this issue on?

Windows

### What architecture is your platform?

64-Bit

### What version of UWB are you using?

Latest VoltstroUPM Release

### What Unity version are you running?

2022.3.10f1

### Describe what the issue you are experiencing is.

Need to set the Zoom level for the browser, using the CEF setting as detailed on the API as follows:
https://cefsharp.github.io/api/101.0.x/html/M_CefSharp_WebBrowserExtensions_SetZoomLevel_1.htm#!

Also, in future, other similar "WebBrowserExtensions" set or get calls should be able to be made. How would these be passed through the UWB interface?

### Provide reproducible steps for this issue.

Integrate UWB and show any website. Now try to set Zoom level so that the display is scaled to a level more appropriate for a user, for example, imagine a UWB output on part of a 4K display in Unity. How can we zoom in to, say, 120% size to make the controls (e.g. scrollbar) larger and easier to interact with, and text bigger and easier to read?

### Any additional info you like to provide?

Nothing specific.

**Comment by Voltstro** [comment]
Moved to discussions. This is a feature request.

I've added both Set/Get Zoom Level in 04a71ba. You can use that commit on what it looks like to add more methods.

With your high resolution DPI scaling, you will need to figure that out your self. There is some discussion [about it on the cef forums](https://magpcss.org/ceforum/viewtopic.php?f=6&t=11491&p=34108). 

Also side note, this project uses CefGlue, not CefSharp. CefGlue's API is much closer to CEF. As an example, the `WebBrowserExtensions` doesn't exist in either CEF or CefGlue, instead SetZoomLevel is on `CefBrowserHost.SetZoomLevel`.
